File: WEKEE.BE/Product.Domain/Dto/CreateProductDtos.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Product.Domain.Dto
{
    public class CreateProductDtos
    {
        public ProductDtos ProductDto { get; set; }
        public List<FeatureProductDtos> FeatureProductDtos { get; set; }
        public List<ImageProductDtos> ImageProductDtos { get; set; }
        public List<HighlightProductDtos> HighlightProductDtos { get; set; }
    }
}

File: WEKEE.FE/UI.HOME/src/scenes/Store/SuNewProduct/dtos/createProductDtos.ts
import { ImageProductDtos } from "./imageProductDtos";
import { ProductDtos } from "./productDtos";

export interface CreateProductDtos {
  productDto: ProductDtos;
  //featureProductDtos: FeatureProductDtos[];
  imageProductDtos: ImageProductDtos[];
  //highlightProductDtos: HighlightProductDtos[];
}
